stdlib/public/core/NewtypeWrapper.swift

```diff
@@ -24,6 +24,9 @@ extension _SwiftNewtypeWrapper where Self.RawValue : Hashable {
 
 #if _runtime(_ObjC)
 extension _SwiftNewtypeWrapper where Self.RawValue : _ObjectiveCBridgeable {
+  // Note: This is the only default typealias for _ObjectiveCType, because
+  // constrained extensions aren't allowed to define types in different ways.
+  // Fortunately the others don't need it.
   public typealias _ObjectiveCType = Self.RawValue._ObjectiveCType
```

Comment:
I don't trust the compiler to get this right consistently, because a `typealias` with the same name as an associated type, but in a constrained extension, isn't something modeled well in the conformance checker. You're already adding a synthesized `typealias` in the importer, so why do you need this?

Reply:
This is the one case the importer is skipping; I didn't want to have to resolve the `_ObjectiveCType` of the `RawValue`, especially when compiling an overlay where we may not have actually determined it yet. How bad do you think this is? (Note that it's not necessarily the original type; a `swift_wrapper` of an `int` is bridged to NSNumber.)

Reply:
I was thinking the that the `swift_bridge` attribute had all of the information, but of course it doesn't for `NSNumber` or `NSValue`. Slightly concerned that we'll be unable to form this type witness if the Clang importer imports one of these types after the type checker is gone, but I supposed https://github.com/apple/swift/pull/15254 should make that no longer an issue.

Reply:
I think if that happens we're hosed anyway for all the default *value* requirements, so I'm not going to worry about it. :-)

Reply:
Associated type witnesses get used in `Type::subst` calls a lot, and we've had a number of compiler crashes due to missing associated type witnesses in the importer. Value witnesses are less often used---only really by the specializer or (in this case) the SILGen bridging code---and hasn't been as much of an issue. Anyway, I'm okay with this solution.